src/app/beer_garden/api/http/handlers/authorization_handler.py

```diff
@@ -172,63 +178,23 @@ def _anonymous_superuser(self) -> "User":
 
         return anonymous_superuser
 
-    def _get_user_from_request(self) -> "User":
```

Comment:
A modified version of this is now in `authentication/__init__.py` so that it could be reused in the `EventSocket` handler.